src/components/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+class Button extends React.Component {
+    constructor(props) {
+        super(props);
+
+    }
+
+    render() {
```

Comment:
Je kan hier `const {color, large, icon, text, link} = this.props` plaatsen om vervolgens alles zonder this.props aan te spreken